turbo.json

```
@@ -16,8 +16,10 @@
       "persistent": true
     },
     "storybook:build": {
-      "dependsOn": ["@webstudio-is/icons#build"],
-      "cache": false
+      "dependsOn": ["@webstudio-is/icons#build", "build"]
```

This will probably work like this:
```suggestion
      "dependsOn": ["build"]
```